src/ol3cesium.js

```diff
@@ -159,7 +159,7 @@ olcs.OLCesium = function(options) {
       this.scene_.initializeFrame();
       this.handleResize_();
       this.dataSourceDisplay_.update(time);
```

Comment:
The time passed to dataSourceDisplay is also incorrect.
We should create a julianDate and pass it to both the dataSourceDisplay and the scene.
```
var julianDate = Cesium.JulianDate.now();
this.dataSourceDisplay_.update(julianDate);
this.scene_.render(julianDate);
```

This should be added to Cesium.externs.js
```
/**
 * @return {Cesium.JulianDate}
 */
Cesium.JulianDate.now = function() {};
```